src/WebJobs.Script/ScriptHostBuilderExtensions.cs

```diff
@@ -43,15 +48,18 @@ public static IHostBuilder AddScriptHost(this IHostBuilder builder, IOptions<Scr
             // Host configuration
             builder.ConfigureLogging((context, loggingBuilder) =>
             {
+                string loggingPath = ConfigurationPath.Combine(ConfigurationSectionNames.JobHost, "Logging");
+                loggingBuilder.AddConfiguration(context.Configuration.GetSection(loggingPath));
+
                 loggingBuilder.Services.AddSingleton<ILoggerProvider, HostFileLoggerProvider>();
                 loggingBuilder.Services.AddSingleton<ILoggerProvider, FunctionFileLoggerProvider>();
 
                 if (ConsoleLoggingEnabled(context))
                 {
                     loggingBuilder.AddConsole(c => { c.DisableColors = false; });
-                    loggingBuilder.SetMinimumLevel(LogLevel.Trace);
-                    loggingBuilder.AddFilter(f => true);
                 }
+
+                ConfigureApplicationInsights(context, loggingBuilder);
```

Comment:
We need to call the registered `IConfigureBuilder<ILoggingBuilder>` here , right?

Reply:
Not here -- that happens in `AddWebScriptHost()`. This is in WebJobs.Script, while the `IConfigureBuilder<>` lives in WebJobs.Script.WebHost

Reply:
👍